### PR TITLE
Enhancement

### PR DESCRIPTION
**Pull request by Rafaelgfirmino**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | 
| License       | MIT
| Doc PR        |

Hey man.
In this pull request I'm doing formatting textArea when the format is json , turning it into an expanded json .

I took for to block the horizontal resize, he always went beyond the layout the profile symfony When resized horizontally.

sorry my poor English!

# Result

### Old Format example
 ```json
{"id":13593847,"name":"GuzzleBundle","full_name":"8p/GuzzleBundle","owner":{"login":"8p","id":767977,"avatar_url":"https://avatars.githubusercontent.com/u/767977?v=3","gravatar_id":"","url":"https://api.github.com/users/8p","html_url":"https://github.com/8p","followers_url":"https://api.github.com/users/8p/followers","following_url":"https://api.github.com/users/8p/following{/other_user}","gists_url":"https://api.github.com/users/8p/gists{/gist_id}","starred_url":"https://api.github.com/users/8p/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/8p/subscriptions","organizations_url":"https://api.github.com/users/8p/orgs","repos_url":"https://api.github.com/users/8p/repos","events_url":"https://api.github.com/users/8p/events{/privacy}","received_events_url":"https://api.github.com/users/8p/received_events","type":"Organization","site_admin":false},"private":false,"html_url":"https://github.com/8p/GuzzleBundle","description":"Integrates Guzzle into Symfony","fork":false,"url":"https://api.github.com/repos/8p/GuzzleBundle","forks_url":"https://api.github.com/repos/8p/GuzzleBundle/forks","keys_url":"https://api.github.com/repos/8p/GuzzleBundle/keys{/key_id}","collaborators_url":"https://api.github.com/repos/8p/GuzzleBundle/collaborators{/collaborator}","teams_url":"https://api.github.com/repos/8p/GuzzleBundle/teams","hooks_url":"https://api.github.com/repos/8p/GuzzleBundle/hooks","issue_events_url":"https://api.github.com/repos/8p/GuzzleBundle/issues/events{/number}","events_url":"https://api.github.com/repos/8p/GuzzleBundle/events","assignees_url":"https://api.github.com/repos/8p/GuzzleBundle/assignees{/user}","branches_url":"https://api.github.com/repos/8p/GuzzleBundle/branches{/branch}","tags_url":"https://api.github.com/repos/8p/GuzzleBundle/tags","blobs_url":"https://api.github.com/repos/8p/GuzzleBundle/git/blobs{/sha}","git_tags_url":"https://api.github.com/repos/8p/GuzzleBundle/git/tags{/sha}","git_refs_url":"https://api.github.com/repos/8p/GuzzleBundle/git/refs{/sha}","trees_url":"https://api.github.com/repos/8p/GuzzleBundle/git/trees{/sha}","statuses_url":"https://api.github.com/repos/8p/GuzzleBundle/statuses/{sha}","languages_url":"https://api.github.com/repos/8p/GuzzleBundle/languages","stargazers_url":"https://api.github.com/repos/8p/GuzzleBundle/stargazers","contributors_url":"https://api.github.com/repos/8p/GuzzleBundle/contributors","subscribers_url":"https://api.github.com/repos/8p/GuzzleBundle/subscribers","subscription_url":"https://api.github.com/repos/8p/GuzzleBundle/subscription","commits_url":"https://api.github.com/repos/8p/GuzzleBundle/commits{/sha}","git_commits_url":"https://api.github.com/repos/8p/GuzzleBundle/git/commits{/sha}","comments_url":"https://api.github.com/repos/8p/GuzzleBundle/comments{/number}","issue_comment_url":"https://api.github.com/repos/8p/GuzzleBundle/issues/comments{/number}","contents_url":"https://api.github.com/repos/8p/GuzzleBundle/contents/{+path}","compare_url":"https://api.github.com/repos/8p/GuzzleBundle/compare/{base}...{head}","merges_url":"https://api.github.com/repos/8p/GuzzleBundle/merges","archive_url":"https://api.github.com/repos/8p/GuzzleBundle/{archive_format}{/ref}","downloads_url":"https://api.github.com/repos/8p/GuzzleBundle/downloads","issues_url":"https://api.github.com/repos/8p/GuzzleBundle/issues{/number}","pulls_url":"https://api.github.com/repos/8p/GuzzleBundle/pulls{/number}","milestones_url":"https://api.github.com/repos/8p/GuzzleBundle/milestones{/number}","notifications_url":"https://api.github.com/repos/8p/GuzzleBundle/notifications{?since,all,participating}","labels_url":"https://api.github.com/repos/8p/GuzzleBundle/labels{/name}","releases_url":"https://api.github.com/repos/8p/GuzzleBundle/releases{/id}","deployments_url":"https://api.github.com/repos/8p/GuzzleBundle/deployments","created_at":"2013-10-15T15:24:34Z","updated_at":"2016-06-30T20:38:10Z","pushed_at":"2016-06-18T14:00:40Z","git_url":"git://github.com/8p/GuzzleBundle.git","ssh_url":"git@github.com:8p/GuzzleBundle.git","clone_url":"https://github.com/8p/GuzzleBundle.git","svn_url":"https://github.com/8p/GuzzleBundle","homepage":"","size":436,"stargazers_count":69,"watchers_count":69,"language":"PHP","has_issues":true,"has_downloads":true,"has_wiki":true,"has_pages":false,"forks_count":20,"mirror_url":null,"open_issues_count":8,"forks":20,"open_issues":8,"watchers":69,"default_branch":"master","organization":{"login":"8p","id":767977,"avatar_url":"https://avatars.githubusercontent.com/u/767977?v=3","gravatar_id":"","url":"https://api.github.com/users/8p","html_url":"https://github.com/8p","followers_url":"https://api.github.com/users/8p/followers","following_url":"https://api.github.com/users/8p/following{/other_user}","gists_url":"https://api.github.com/users/8p/gists{/gist_id}","starred_url":"https://api.github.com/users/8p/starred{/owner}{/repo}","subscriptions_url":"https://api.github.com/users/8p/subscriptions","organizations_url":"https://api.github.com/users/8p/orgs","repos_url":"https://api.github.com/users/8p/repos","events_url":"https://api.github.com/users/8p/events{/privacy}","received_events_url":"https://api.github.com/users/8p/received_events","type":"Organization","site_admin":false},"network_count":20,"subscribers_count":5}
 ```

### New Format example
 ```json
{
  "id": 13593847,
  "name": "GuzzleBundle",
  "full_name": "8p/GuzzleBundle",
  "owner": {
    "login": "8p",
    "id": 767977,
    "avatar_url": "https://avatars.githubusercontent.com/u/767977?v=3",
    "gravatar_id": "",
    "url": "https://api.github.com/users/8p",
    "html_url": "https://github.com/8p",
    "followers_url": "https://api.github.com/users/8p/followers",
    "following_url": "https://api.github.com/users/8p/following{/other_user}",
    "gists_url": "https://api.github.com/users/8p/gists{/gist_id}",
    "starred_url": "https://api.github.com/users/8p/starred{/owner}{/repo}",
    "subscriptions_url": "https://api.github.com/users/8p/subscriptions",
    "organizations_url": "https://api.github.com/users/8p/orgs",
    "repos_url": "https://api.github.com/users/8p/repos",
    "events_url": "https://api.github.com/users/8p/events{/privacy}",
    "received_events_url": "https://api.github.com/users/8p/received_events",
    "type": "Organization",
    "site_admin": false
  },
......
}
```

### CODE REVIEW

File: Resources/views/profiler.html.twig [hunk 1]
```diff
@@ -19,7 +19,8 @@
                     <a href="javascript:;">
                         <span class="method">{{ message.request.method }}</span>
                         <span class="url">{{ message.request.url }}</span>
-                        <span class="response">{{ message.response.statusPhrase }} ({{ message.response.statusCode }})</span>
+                        <span class="response">{{ message.response.statusPhrase }} ({{ message.response.statusCode }}
+                            )</span>
```
Comment (florianpreusner) [inline Resources/views/profiler.html.twig]: why splitting into two lines?

File: Resources/views/profiler.html.twig [hunk 2]
```diff
@@ -71,7 +72,7 @@
                             <tr>
                                 <th>Body</th>
                                 <td>
-                                    <textarea>{{ message.response.body }}</textarea>
+                                    <textarea readonly id="resposeBody">{{ message.response.body }}</textarea>
```
Comment (florianpreusner) [inline Resources/views/profiler.html.twig]: typo: respose vs. respo*n*se

File: Resources/views/profiler.html.twig [hunk 3]
```diff
@@ -80,18 +81,34 @@
...
             <script>
-                $("#request_{{ i }} h3 a").click(function() {
+                $("#request_{{ i }} h3 a").click(function () {
                     $("#request_{{ i }} .content").toggle("fast");
                 });
+
+                var requestBody = $('#requestBody');
+                requestBody.val(formatJson(requestBody.val()));
+
+                var responseBody = $('#resposeBody');
```
Comment (florianpreusner) [inline Resources/views/profiler.html.twig]: same typo

**Comment by florianpreusner** [comment]
Awesome! Thanks for contributing!
I found some minor things. Would be nice if you can fix them so I can merge your request afterwords.
Best regards!

**Comment by florianpreusner** [comment]
🎉  Nice! Thanks!

**Comment by Rafaelgfirmino** [comment]
You're welcome!

**Pull request merged**